examples/factor_analysis/README.rst

```diff
@@ -0,0 +1,8 @@
+Examples for TFA/HTFA
+==========================
+
+Install click is requred for running get_tfa_input_from_nifti.py::
+
+    pip install -U click 
```

Comment:
Change to `pip3 install -U -r requirements.txt` and add a `requirements.txt` file.